VERSION

```diff
@@ -1 +1 @@
-0.152.2
+0.152.3
```

Comment:
Since it's a feat, I think it's fine to increase the minor version

Reply:
fixed :)